package.json

```diff
@@ -11,10 +11,9 @@
     "url": "git+https://github.com/Marfeel/flowcards-playground.git"
   },
   "scripts": {
-    "test:local": "npm run appium:tab && END_TO_END_MODE=local npm t",
-    "test:remote-local": "END_TO_END_MODE='browserstack-local' npm t",
-    "test:remote": "END_TO_END_MODE='browserstack-remote' npm t",
-    "test": "wdio conf/conf.js",
+    "test:local": "END_TO_END_MODE=local wdio conf/conf.js",
+    "test:remote-local": "END_TO_END_MODE=browserstack-local wdio conf/conf.js",
+    "test:remote": "END_TO_END_MODE=browserstack-remote wdio conf/local.conf.js",
```

Comment:
💅🏽 I created command `test` to """centralize""" the command in one place so we can apply flags to each of them but if you prefer it like this is fine to me.